### ISSUE TITLE
Increase the amount of configurable buttons

### ISSUE THREAD

**Issue by JeffAtHuemor** [beaverbuilder/feature-requests#133]
Most modern websites contain multiple button styles, the customizer should allow you to specify more than one for the site.

The user could then have the ability to choose from these different button styles at the module level as 'presets'. The GUI for this could be handled via a drop-down or something similar.

**Comment by codepuncher** [comment]
This is a must-have!

CC @ramshengale

**Comment by ramshengale** [comment]
Agreed. Possible solution suggested [here](https://github.com/beaverbuilder/feature-requests/issues/140) .

**Comment by steveniwd** [comment]
I came here to post the same request! Most sites only use 2, maybe 3 button styles. Making button styling a global preset and then setting the button to the preset makes so much more sense than individually doing it, or messing with CSS!

**Comment by codente** [comment]
@steveniwd This is for the bb theme.  Are you referring to the BB theme or the plugin?

**Comment by steveniwd** [comment]
@codente oops! I was talking about the BB Plugin, not the theme. I didn't see this was on the theme.
